pkg/target/target.go

```diff
@@ -426,6 +427,95 @@ func convertToLabelSelector(object map[string]interface{}) (*metav1.LabelSelecto
 	return obj, nil
 }
 
+// TODO: can we use generic for Unmarshal after go 1.18?
+func convertToMatch(object map[string]interface{}) (*match.Match, error) {
+	j, err := json.Marshal(object)
+	if err != nil {
+		return nil, errors.Wrap(err, "Could not convert unknown object to JSON")
+	}
+	obj := &match.Match{}
+	if err := json.Unmarshal(j, obj); err != nil {
+		return nil, errors.Wrap(err, "Could not convert JSON to Match")
```

Comment:
Use lowercase for error strings. So "could not convert" as opposed to "Could not convert"

Reply:
done.